index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Ulises Cappa's Portfolio</title>
+</head>
+<body>
+
+    <header class="heading">
+        <!-- CELPHONE BOX -->
+
+        <div class="phone-box">
+            <div class="time">
+                <time datetime="9:41">9:41</time>
+            </div>
+
+            <div class="cell-info">
+                <img src="Images/CellularConnection.svg" alt="">
+                <img src="Images/Wifi.svg" alt="">
+                <img src="Images/Battery.svg" alt="">
+            </div>
+            
+        </div>
+    
+
+<!-- ########### NAVBAR ############### -->
+        <nav class="nav-header">
+            <!--         LOGO -->
+            <div class="logo">
+                My Logo
+            </div>
+            <!--         MENU -->
+            <div class="menu">
+                <!-- <ul>
+                    <li>About Me</li>
+                    <li>My Projects</li>
+                    <li>Contact Me</li>
+                </ul> -->
+
+                <svg  width="18" height="12" viewBox="0 0 18 12" fill="none" xmlns="http://www.w3.org/2000/svg">
+                    <path fill-rule="evenodd" clip-rule="evenodd" d="M0 1C0 0.447715 0.447715 0 1 0H17C17.5523 0 18 0.447715 18 1C18 1.55228 17.5523 2 17 2H1C0.447715 2 0 1.55228 0 1ZM0 6C0 5.44772 0.447715 5 1 5L17 5C17.5523 5 18 5.44772 18 6C18 6.55229 17.5523 7 17 7L1 7C0.447715 7 0 6.55228 0 6ZM1 10C0.447715 10 0 10.4477 0 11C0 11.5523 0.447715 12 1 12H17C17.5523 12 18 11.5523 18 11C18 10.4477 17.5523 10 17 10H1Z" fill="white"/>
+                </svg>
+                    
+            </div>
+            
+        </nav>
+
+        
+    </header>
+
+    <div class="main">
+        
+        <h1 class="title-name">Hey There. I'm Ulises.</h1>
+        <h2 class="title-about">I'm a Software Developer</h2>
+        
```

Comment:
- [ ] Please make sure that the container of the section is a `section` tag instead of `div` for semantic purposes.  `<section> `means that the content inside is grouped (i.e. relates to a single theme), and should appear as an entry in an outline of the page. `<div>` , on the other hand, does not convey any meaning, aside from any found in its class , lang and title attributes. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.